src/form/iv.jl

```diff
@@ -152,109 +94,67 @@ function constraint_voltage_angle_difference(pm::AbstractIVRModel, n::Int, c::In
     vit = var(pm, n, c, :vi, t_bus)
```

Comment:
nit, `vr_fr`,`vr_to`,`vi_fr`,`vi_to` would be more consistent with the rest of the codebase.

Reply:
ok